Tests/ChessGameTest.php

```diff
@@ -244,6 +244,34 @@ public function testCompleteGame()
         $this->assertEquals($endFen, $this->game->renderFen());
     }
 
+    /**
+     * Test two knights that can both move to the same square, but one is pinned to the king.
+     *
+     * 1.e4 e5 2.Bc4 Nf6 3.d3 Nc6 4.Nc3 Bb4 5.Ne2
+     */
+    public function testAmbiguousKnightMoves()
+    {
+        $endFen = 'r1bqk2r/pppp1ppp/2n2n2/4p3/1bB1P3/2NP4/PPP2PPP/R1BQK1NR w KQkq - 3 5';
```

Comment:
I think that the resulting fen should be this one: r1bqk2r/pppp1ppp/2n2n2/4p3/1bB1P3/2NP4/PPP1NPPP/R1BQK2R b KQkq - 4 5 (after Ne2 or Nge2 was played)